gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/reactive/core/v4/endpoint/DefaultEndpointManager.java

```diff
@@ -250,8 +250,8 @@ private void createAndStartEndpoint(final ManagedEndpointGroup managedEndpointGr
             final ManagedEndpoint managedEndpoint = new DefaultManagedEndpoint(endpoint, managedEndpointGroup, connector);
             managedEndpointGroup.addManagedEndpoint(managedEndpoint);
             endpointsByName.put(endpoint.getName(), managedEndpoint);
-            endpointVariables.put(endpoint.getName(), endpoint.getName() + ":");
-            endpointVariables.put(managedEndpointGroup.getDefinition().getName(), managedEndpointGroup.getDefinition().getName() + ":");
+            endpointVariables.put(endpoint.getName(), endpoint.getName());
+            endpointVariables.put(managedEndpointGroup.getDefinition().getName(), managedEndpointGroup.getDefinition().getName());
```

Comment:
Alternatively, we can add both, one without the ':' and one with so we are sure it does not break any other policy that makes use of the EL `{#endpoints['my-endpoint:']}` (other than the traffic shadowing policy).
WDYT?

Reply:
As seen together, that's not possible cause the ':' is on the value not on the key 🤦🏻‍♂️